### PR TITLE
added a method to get readme 

### PR DESCRIPTION
**Pull request by ayushanand308**
Saves the readme.md of given user to current working directory

### CODE REVIEW

**Comment by ayushanand308** [comment]
@nikhil25803  Please review the PR and tell me if you need any changes.

File: src/scrape_up/github/__init__.py [hunk 1]
```diff
@@ -1,5 +1,130 @@
+import requests
```
Comment (nikhil25803) [inline src/scrape_up/github/__init__.py:1]: @ayushanand308 
Do not make changes to `__init__.py`, there's a separate file created for `Users`, `Repository`, etc. Add this method in the repository class of the `github/repository.py` file.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

File: src/scrape_up/github/__init__.py (hunk 1 above)
Comment (nikhil25803) [reply src/scrape_up/github/__init__.py:1]: Looks like you haven't updated the forked repository, sync it with the current project status and also update in your computer.

**Comment by ayushanand308** [comment]
Did the necessary changes but after it got pushed as a new PR. @nikhil25803 Can you review that instead.

**Review by nikhil25803 — APPROVED** [review]
Closing this PR, will review and merge the new PR @ayushanand308.

**Pull request closed without merging**